src/test/java/com/fontys/api/service/TournamentServiceTest.java

```diff
@@ -1,33 +1,56 @@
 package com.fontys.api.service;
 
-import com.fontys.api.entities.Tournament;
-import com.fontys.api.entities.User;
-import com.fontys.api.repositories.TournamentRepository;
-import com.fontys.api.repositories.UserRepository;
-import org.junit.Before;
-import org.junit.Test;
-import org.mockito.Mockito;
```

Comment:
What happened here?

Reply:
I actually have no idea. But I fixed it now.